DOC-1757: Updates so the user would understand how to update the default options (#2554)
tinymce/tinymce-docs: modules/ROOT/pages/apis/tinymce.root.adoc

----
overrideDefaults(defaultOptions: Object)
----

> `overrideDefaults` will replace any of the defaults set by a previous call to the `overrideDefaults` function. 
> When using the cloud some of the defaults are required for the cloud based editor to function.
> 
> Therefore when using `overrideDefaults` with the cloud based editor, the user will need to combine the previous defaults `tinymce.defaultOptions` with the new options that you want to integrate. 

Reviewer: ```suggestion
`overrideDefaults` replaces any of the defaults set by a previous call to the `overrideDefaults` function.

When using the cloud, some of these defaults are required for the cloud-based editor to function.

Therefore, when using `overrideDefaults` with the cloud-based editor, combine the previous defaults, `tinymce.defaultOptions`, with the new options that you want to integrate.
```
Suggested copy-edits.

Reviewer: Also, is _cloud_ above generic, or is it a reference to the Tiny Cloud?

If the latter, `s/cloud/{cloudname}/` should be run over the new text.
